charts/consul/templates/webhook-cert-manager-clusterrole.yaml

```diff
@@ -40,6 +40,14 @@ rules:
   verbs:
   - get
 {{- if .Values.global.enablePodSecurityPolicies }}
+- apiGroups:
+  - policy
+  resources:
+  - podsecuritypolicies
+  resourceNames:
+  - {{ template "consul.fullname" . }}-webhook-cert-manager
+  verbs:
+  - use
 - apiGroups:
   - policy
```

Comment:
You can just rename the connect-injector to be the webhook-cert-manager. It being connect-injector is a bug!